robottelo/cli/base.py

```diff
@@ -354,7 +356,8 @@ def info(cls, options=None, output_format=None):
         return result
 
     @classmethod
-    def list(cls, options=None, per_page=True, output_format='csv'):
+    def list(cls, options=None, per_page=True,  credentials=True,
```

Comment:
Well, I thought the same thing, base.py should be refactored to be more flexible, how about moving the discussion to a separate issue. I wonder if we could use kwargs in this scenario.